src/main/java/gregtech/common/ClientProxy.java

```diff
@@ -158,6 +160,17 @@ public static void registerModels(ModelRegistryEvent event) {
         MetaItems.registerModels();
     }
 
+    @SubscribeEvent
+    public static void registerSprites(TextureStitchEvent.Pre event) {
+        for (MaterialIconSet set : MaterialIconSet.ICON_SETS.values()) {
+            event.getMap().registerSprite(MaterialIconType.ore.getBlockPath(set));
+            event.getMap().registerSprite(MaterialIconType.block.getBlockPath(set));
+        }
+        MetaBlocks.COMPRESSED.values().stream().distinct().forEach(c -> c.onTextureStitch(event));
+        MetaBlocks.FRAMES.values().stream().distinct().forEach(f -> f.onTextureStitch(event));
+        MetaBlocks.ORES.forEach(o -> o.onTextureStitch(event));
+    }
```

Comment:
This theoretically could be automated better, but since we only have 3 custom baked models, this is fine as is. Could be improved when we start making surface rock models or something.